src/test/ui/seq-args.stderr

```diff
@@ -1,26 +1,26 @@
-error[E0107]: this trait takes 0 type arguments but 1 type argument was supplied
+error[E0107]: this trait takes 0 generic arguments but 1 generic argument was supplied
   --> $DIR/seq-args.rs:4:13
    |
 LL |     impl<T> Seq<T> for Vec<T> {
    |             ^^^--- help: remove these generics
    |             |
-   |             expected 0 type arguments
+   |             expected  0 generic arguments
```

Comment:
It seems like a change to the formatting strings is introducing an extra unnecessary space.

Reply:
Oh, I'll change that.

Reply:
I believe this is still in need of attention.